apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -393,6 +392,9 @@ const Ticket = new GQLListSchema('Ticket', {
                 relatedManyToManyResolvers,
                 [{ property, unitName }, { placeClassifier, categoryClassifier, problemClassifier }]
             )(...args)
+
+            /* NOTE: this sends different kinds of notifications on ticket create/update */
+            await triggerTicketEventActions(args[0])
```

Comment:
Destructure, please `args` to make it clear, what the first element is.

Reply:
done